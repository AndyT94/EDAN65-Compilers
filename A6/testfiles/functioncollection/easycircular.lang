int main() {
	b();
}

int b() {
	a();
}

int a() {
	b();
}