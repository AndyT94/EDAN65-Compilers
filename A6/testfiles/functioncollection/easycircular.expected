main -> b
b -> a
a -> b
b is circular
a is circular
